package.json

```diff
@@ -36,19 +37,22 @@
 	"scripts": {
 		"build": "esbuild src/content-script.ts --bundle --minify --outfile=lib/content-script.js --sourcemap",
 		"compile": "tsc",
+		"firefox:dev": "web-ext run",
+		"firefox:package": "web-ext build",
 		"format": "prettier \"**/*\" --ignore-unknown",
 		"format:write": "yarn format --write",
 		"lint": "npx eslint \"src/*.ts\"  --max-warnings 0 --report-unused-disable-directives",
 		"lint:md": "npx markdownlint \"**/*.md\" \".github/**/*.md\"",
 		"lint:spelling": "cspell \"**\" \".github/**/*\"",
 		"prepare": "husky install",
+		"webext:lint": "web-ext lint",
```

Comment:
[Tooling] Any new building or linting should have a corresponding GitHub Actions workflow. That way CI validates nobody breaks them! See existing `compile.yml` & others.